core/cloudflow-operator/src/main/scala/cloudflow/operator/event/ConfigInputChangeEvent.scala

```diff
@@ -156,24 +156,29 @@ object ConfigInputChangeEvent {
 
   def getConfigFromSecret(configInputChangeEvent: ConfigInputChangeEvent, system: ActorSystem) = {
     val secret: Secret = configInputChangeEvent.watchEvent._object
+    val empty          = ConfigFactory.empty()
     secret.data
       .get(ConfigInputChangeEvent.SecretDataKey)
-      .map { bytes =>
-        Try(ConfigFactory.parseString(new String(bytes, StandardCharsets.UTF_8))).toOption.getOrElse {
-          system.log.error(
-            s"Detected input secret '${secret.metadata.name}' contains invalid configuration data, IGNORING configuration."
-          )
-          ConfigFactory.empty()
-        }
+      .flatMap { bytes =>
+        val str = new String(bytes, StandardCharsets.UTF_8)
+        Try(ConfigFactory.parseString(str).resolve()).recover {
+          case cause =>
+            system.log.error(
+              cause,
+              s"Detected input secret '${secret.metadata.name}' contains invalid configuration data, IGNORING configuration."
```

Comment:
There is nothing we can do to hint whats invalid?

Reply:
The cause is there?